py/strato/common/log/log2text.py

```diff
@@ -233,6 +235,53 @@ def updateConfFile(field, value):
         f.write(yaml.dump(conf))
     os.system('sudo mv /tmp/stratolog-conf.tmp %s' % LOG_CONFIG_FILE_PATH)
 
+
+class LogPathFinder:
+    def __init__(self, confFile):
+        try:
+            self.confFile = yaml.load(open(confFile, 'r').read())
+        except:
+            pass
+        self.defaultPaths = []
+
+    def findLogFiles(self, providedPaths):
+        logsToRead = []
+        [logsToRead.append(filePath) if os.path.isfile(filePath) else logsToRead.extend(self._tryToMatchShortcut(filePath)) for filePath in providedPaths]
+        return logsToRead
+
+    def _tryToMatchShortcut(self, filePath):
+        try:
+            if filePath in self.confFile['defaultBundles']:
+                return self.matchBundle(filePath)
+            else:
+                return self.tryToMatchDefaultPath(filePath)
+        except:
+            self._raiseNoMatchError(filePath)
+
+    def matchBundle(self, filePath):
+        filesInBundle = glob.glob(self.confFile['defaultBundles'][filePath])
+        if filesInBundle:
+            return filesInBundle
+        else:
+            self._raiseNoMatchError(filePath)
+
+    def tryToMatchDefaultPath(self, filePath):
+        if not self.defaultPaths:
+            self._getDefaultPaths()
+        matchingFiles = [defaultFilePath for defaultFilePath in self.defaultPaths
+                         if filePath.lower() in defaultFilePath.lower() and os.path.isfile(defaultFilePath)]
+        if not matchingFiles:
+            self._raiseNoMatchError(filePath)
+        else:
+            return matchingFiles
+
+    def _raiseNoMatchError(self, filePath):
+        raise Exception("No matching files were found for %s" % filePath)
+
+    def _getDefaultPaths(self):
```

Comment:
I think I lost the flow here. You have something in init, than you have default from another place... why do you need al this mismatch?